app/views/catalog/_zero_results.html.erb

```diff
@@ -2,7 +2,11 @@
   <h2 class="h1 text-center"><%= t 'blacklight.search.zero_results.title' %></h2>
   <div id="documents" class="noresults text-center">
     <h3 class="h5 mw-lg"><i><%= t 'blacklight.search.zero_results.modify_search' %>:</i></h3>
-    <p class="mw font-sm"> <%= t 'blacklight.search.zero_results.use_fewer_keywords' %> </p>
+    <p class="mw font-sm"> <%= t 'blacklight.search.zero_results.variant_spelling' %> </p>
```

Comment:
Try `mw-sm` instead of `font-sm`, particularly for the contact us line. The link text needs to stand out more for a11y, it should get an underline with `mw-sm`